src/commcare_cloud/ansible/roles/formplayer/templates/application.properties.j2

```diff
@@ -51,3 +51,7 @@ redis.Password={{ localsettings.REDIS_PASSWORD }}
 endpoints.enabled=true
 management.address: 0.0.0.0
 management.port: 8081
+
+{% if formplayer_sensitive_data_logging %}
+sensitiveData.enableLogging=true
+{% endif %}
```

Comment:
Nit: should be a newline at the end of the file. I recommend configuring your editor to do this for you. (For pycharm you can check out this SO answer: https://stackoverflow.com/a/37260222)